<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImageRGB</class>
 <widget class="QWidget" name="ImageRGB">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>590</width>
    <height>375</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>590</width>
    <height>375</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>590</width>
    <height>375</height>
   </size>
  </property>
  <property name="sizeIncrement">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>590</width>
    <height>375</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Microsoft YaHei UI</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>图片像素提取</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="1,0,9">
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0">
      <item>
       <widget class="QLineEdit" name="lineEdit">
        <property name="dragEnabled">
         <bool>true</bool>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="OpenImageBtn">
        <property name="baseSize">
         <size>
          <width>0</width>
          <height>19</height>
         </size>
        </property>
        <property name="text">
         <string>打开图片</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="PreImage">
      <attribute name="title">
       <string>像素提取</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,1">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QRadioButton" name="radioButton_1">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>100*100像素</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="radioButton_2">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>50*50像素</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="radioButton">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>原尺寸像素</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="GetRGBBtn">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="baseSize">
            <size>
             <width>0</width>
             <height>18</height>
            </size>
           </property>
           <property name="text">
            <string>开始提取像素</string>
           </property>
           <property name="checkable">
            <bool>false</bool>
           </property>
           <property name="autoDefault">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QFrame" name="frame_3">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QGroupBox" name="groupBox">
            <property name="title">
             <string>原图预览</string>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_5">
             <item>
              <widget class="QLabel" name="orgin">
               <property name="minimumSize">
                <size>
                 <width>165</width>
                 <height>165</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>165</width>
                 <height>165</height>
                </size>
               </property>
               <property name="baseSize">
                <size>
                 <width>165</width>
                 <height>165</height>
                </size>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBox_2">
            <property name="title">
             <string>100*100</string>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_6">
             <item>
              <widget class="QLabel" name="thumb_100">
               <property name="minimumSize">
                <size>
                 <width>165</width>
                 <height>165</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>165</width>
                 <height>165</height>
                </size>
               </property>
               <property name="baseSize">
                <size>
                 <width>165</width>
                 <height>165</height>
                </size>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBox_3">
            <property name="title">
             <string>50*50</string>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_7">
             <item>
              <widget class="QLabel" name="thumb_50">
               <property name="minimumSize">
                <size>
                 <width>165</width>
                 <height>165</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>165</width>
                 <height>165</height>
                </size>
               </property>
               <property name="baseSize">
                <size>
                 <width>165</width>
                 <height>165</height>
                </size>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="RGBtoHEX">
      <attribute name="title">
       <string>像素文本</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QTextBrowser" name="textBrowser">
         <property name="readOnly">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
